Garden-Service/src/Garden-Service.WebAPI/Controllers/GardenController.cs

```diff
@@ -21,6 +21,12 @@ public class GardenController : ControllerBase {
 		return Ok(createGardenResponse);
 	}
 
+	[HttpPut(Name = "update_garden")]
+	public IActionResult UpdateGarden(EditGardenRequest request) {
```

Comment:
Es ergibt keinen Sinn, asynchrone Service-Methoden zu erstellen, welche dann im Controller auf Krampf sync ausgeführt werden. Das Ergebnis ist eine negativer Overhead, dank der TPL. Bitte Controller ebenfalls Async machen, falls möglich.

Reply:
Neue version bitte überprüfenNeue version bitte überprüfen. Ist jetzt ähnlich wie in IdentityService gehalten